documentation(Office365/MessageTrace): Add `start_time` documentation. (#526)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/cloud_and_saas/office365/message_trace.md

Save your configuration and start the trigger.

## Configure time range

> Configure the trigger `timedelta` parameter to pull only events from 24 hours ago (1440 minutes) to be compliant with [Microsoft documentation](https://learn.microsoft.com/en-us/previous-versions/office/developer/o365-enterprise-developers/jj984335(v=office.15)?redirectedfrom=MSDN#data-granularity-persistence-and-availability):

Reviewer: ```suggestion
Configure the trigger `timedelta` parameter to shift events pulling. The maximum timedelta is 24 hours ago (1440 minutes) to be compliant with [Microsoft documentation](https://learn.microsoft.com/en-us/previous-versions/office/developer/o365-enterprise-developers/jj984335(v=office.15)?redirectedfrom=MSDN#data-granularity-persistence-and-availability):
```

> Events may be delayed by up to 24 hours before they appear in a report.

Configure the trigger `start_time` parameter to pull events from X hours ago if you want to import events that happened in the past. You can, for example, pull events from 30 days ago to now as explain in [Microsoft documentation](https://learn.microsoft.com/en-us/previous-versions/office/developer/o365-enterprise-developers/jj984335(v=office.15)?redirectedfrom=MSDN#data-granularity-persistence-and-availability):

> > The information for this report is available for a period of 30 days, or until the subscription is canceled.

Reviewer: ```suggestion
> The information for this report is available for a period of 30 days, or until the subscription is cancelled.
```
